captcha-validation-service/pom.xml

```diff
@@ -110,12 +127,12 @@
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-ref-idobjectvalidator</artifactId>
-					<version>1.2.0-rc1</version>
+					<version>1.2.0.1-B2</version>
 				</dependency>
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-transliteration-icu4j</artifactId>
-					<version>1.2.0-rc1</version>
+					<version>1.2.0.1-B2</version>
```

Comment:
Point to migrated version. Migrate it if it is not yet migrated.

Reply:
Done with the changes